docs/PackageManagement/Add-AatPackageModule.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: AzureAutomationTools.PackageManagement-help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Add-AatPackageModule
+
+## SYNOPSIS
+Adds a module to a module package file.
+
+## SYNTAX
+
+### WithoutPackage (Default)
```

Comment:
This will break all existing module files (I was aiming to have this just slot into place), but I'm happy to work this if you still wish.